charts/platform-tenant/templates/_ingressgateway.tpl

```diff
@@ -0,0 +1,21 @@
+{{/* 
```

Comment:
This file should be named `_ingress-route.tpl` prob and use that variable, not `ingressgateway`